amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2091,7 +2094,148 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/events':
+    get:
+      tags:
+        - events
+      summary: List all the record management events
+      description: |
+        Use this to get all the events in the record management site. This return all the custom events as well as default events in record management site with mandatory fields and properties
+
+      operationId: getAllEvents
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventsInfo'
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+
+    post:
+      tags:
+        - events
+      summary: Create a new records management event
+      description: |
+        Create a new records management event.
+      operationId: createEvent
+      parameters:
+        - in: body
+          name: eventBodyCreate
+          description: The event information to create. The event type value can be one of the following- [ Simple, Obsoleted, Superseded, Cross-Referenced Record Transferred, Versioned ]
+          required: true
+          schema:
+            $ref: '#/definitions/EventBodyCreate'
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      responses:
+        '201':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/EventEntry'
+        '400':
+          description: Cannot create event. An event with the name **name** already exists  
+        '401':
+          description: Authentication failed
```

Comment:
I think there should be a 403 error here too.

Reply:
Done

Reply:
Not done?

Reply:
Done